src/libs/actions/User.js

```diff
@@ -232,15 +232,19 @@ function validateLogin(accountID, validateCode) {
 /**
  * Checks if the expiresAt date of a user's ban is before right now
```

Comment:
Let's update this comment to indicate that it checks the blockedFromConcierge object to see if it has an expiresAt key and if so whether the expiresAt date of a user's ban is before right now

Reply:
Done.